lib/currency.rb

```diff
@@ -0,0 +1,29 @@
+module Currency
+  extend self
+  
+  def args(*currency)
+    currency = currency.flatten.compact
```

Comment:
Why would you tolerate a drunk programmer? Who being in clear mind would pass `nil`s there?